stylus/ui-components/modals.styl

```diff
@@ -34,7 +34,6 @@ $modals
         display           block
         box-sizing        border-box
         border-radius     em(8px)
```

Comment:
Sorry, i meant it would take full width if the content pushes to the edges.
I think a `max-width: 90%` just to secure some margin from the edges would be better.